auth/src/main/resources/application.yml

```diff
@@ -1,26 +1,28 @@
 spring:
   application:
     name: "auth"
-  profiles:
-    active: local
   security:
     oauth2:
       resourceserver:
         opaquetoken: # Set client secret in SPRING_SECURITY_OAUTH2_RESOURCESERVER_OPAQUETOKEN_CLIENT_SECRET env variable
           client-id: 0oaek8tip2lhrhHce1d7
           introspection-uri: https://reportstream.oktapreview.com/oauth2/ausekaai7gUuUtHda1d7/v1/introspect
-  cloud:
-    gateway:
-      proxy:
-        sensitive: [] # pass authorization and cookie headers downstream (filtered by default)
 
 server.port: 9000
 
-proxy.pathMappings:
-  - pathPrefix: /reportstream
-    baseUrl: http://localhost:7071
-  - pathPrefix: /submissions
-    baseUrl: http://localhost:8880
+app:
+  environment: local
+
+# submissions microservice configuration
+submissions:
+  baseUrl: http://localhost:8080
+
+# Ensure these are disabled in production
+springdoc:
```

Comment:
I'm not sure what we're getting out of the swagger docs, what are the other endpoints that would ultimately be here?

Reply:
It would be more helpful to write an openapi spec for the URLs were expecting and have swagger load that.

Reply:
Ya the thought was we would document the APIs this proxy supports in the proxy itself since the other endpoints will not be available to the outside world

Reply:
I just found this article which would be nice way to handle this!

https://medium.com/@erkndmrl/swagger-3-with-spring-cloud-gateway-a-comprehensive-guide-f355459d914a

tl;dr Setting up swagger on downstream services and configuring the auth project to load their specs